readers/influxdb/messages_test.go

```diff
@@ -2,28 +2,29 @@ package influxdb_test
 
 import (
 	"fmt"
-	"os"
 	"testing"
 	"time"
 
 	influxdata "github.com/influxdata/influxdb/client/v2"
 	"github.com/mainflux/mainflux/pkg/transformers/senml"
+	uuidProvider "github.com/mainflux/mainflux/pkg/uuid"
 	"github.com/mainflux/mainflux/readers"
-	reader "github.com/mainflux/mainflux/readers/influxdb"
-	writer "github.com/mainflux/mainflux/writers/influxdb"
+	influxReader "github.com/mainflux/mainflux/readers/influxdb"
```

Comment:
Package names are usually simple and short(`ioutil`, `strconv`) with no underscores and mixed caps (https://blog.golang.org/package-names), and package names in the import sections are no different.